src/test/java/com/google/sps/servlets/DataServletTest.java

```diff
@@ -171,6 +178,235 @@ public void addTermRating_overwritingExistingTermRating() throws IOException {
     assertEquals((double) -0.699999988079071, termRatingEntity.getProperty("score-professor"));
   }
 
+  @Test
+  public void addTermRating_WithTranslation() throws IOException {
+    // SETUP.
+    // File with body request in webapp folder.
+    HttpServletRequest request = Mockito.mock(HttpServletRequest.class);
+    Reader reader = new FileReader("src/main/webapp/WEB-INF/testTranslation.txt");
+    BufferedReader bufferedReader = new BufferedReader(reader);
+    when(request.getReader()).thenReturn(bufferedReader);
+
+    AnalyzeSentimentResponse response =
+        AnalyzeSentimentResponse.newBuilder()
+            .setDocumentSentiment(
+                Sentiment.newBuilder()
+                    .setScore((float) -0.8999999761581421)) // Sentiment Score of Text.
```

Comment:
This float is pretty long and arbitrary. Might want to have it as a const rather than typing it out like this each time it's used.

Reply:
Just put it in a variable, for some reason when I try to change it to a double from a float it shortens the number itself. Hence, when I am doing assert I am typing it out in the asserts.